tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -538,7 +531,9 @@ describe("Service Form Modal", function() {
         cy.get(".modal-full-screen-side-panel.is-visible").then(function(
           $jsonEditor
         ) {
-          expect($jsonEditor.width()).to.equal(500);
+          expect($jsonEditor.width()).to.equal(
+            Cypress.config("viewportWidth") / 2
```

Comment:
This is the only real change in this file. The rest is from my text editor running Prettier on save